src/components/tables/DetailsPanel.tsx

```diff
@@ -0,0 +1,43 @@
+import { Collapse, TableCell, TableRow } from '@mui/material';
+import CaptureDetails from 'components/capture/Details';
+import { createEditorStore } from 'components/editor/Store';
+import { ZustandProvider } from 'hooks/useZustand';
+
+interface Props {
+    detailsExpanded: boolean;
+    id: string;
+    storeName?: string;
+    colSpan: number;
+    disableLogs?: boolean;
+}
+
+function DetailsPanel({
```

Comment:
Putting details into component so other tables can use it. Still letting the parent handle the state stuff.